plugins/hydra_nevergrad_sweeper/hydra_plugins/hydra_nevergrad_sweeper/config.py

```diff
@@ -78,9 +80,6 @@ class NevergradSweeperConf:
     # - as a full scalar specification
     parametrization: Dict[str, Any] = field(default_factory=dict)
 
-    # version of the commandline API
-    version: int = 1
-
```

Comment:
Good.
Was this removed from the code too?

Reply:
yes this is removed :)